### PR TITLE
fix(documentation): Fix typos in English documentation

### PR DESCRIPTION
**Pull request by joyhchen**
This mostly fixes typos, but there are also some minor grammatical changes

### CODE REVIEW

**Comment by susiwen8** [comment]
Thank you very much!!!!!!!!

File: en/option/component/visual-map-piecewise.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ When they are not specified, `itemSymbol` is adopted as the default value (but j
 
 the formatter tool for label.
 
-+ If it was set as a `string`, it refers to a template, for instance: `aaaa{value}bbbb{value2}`, where `{value}` and `{value2}` represents the current seleted range of dataValues.
++ If it was set as a `string`, it refers to a template, for instance: `aaaa{value}bbbb{value2}`, where `{value}` and `{value2}` represents the current selected range of dataValues.
```
Comment (plainheart) [inline en/option/component/visual-map-piecewise.md:218]: > If it was set as a `string`, it refers to a template,

No need to use `was`, use `is` instead.

File: en/option/series/map.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 **Map.**
 
-Map is maily used in the visulization of geographic area data, which can be used with [visualMap](~visualMap) component to visualize the datas such as population distribution density in diffrent areas.
+Map is maimly used in the visualization of geographic area data, which can be used with [visualMap](~visualMap) component to visualize the datas such as population distribution density in diffrent areas.
```
Comment (plainheart) [inline en/option/series/map.md:8]: > Map is maimly used in the visualization

mainly  -> mainly?

Comment (Ovilia) [reply en/option/series/map.md:8]: Oh. This is a typo I didn't notice.

**Comment by plainheart** [comment]
That's OK. This PR can be merged but need to correct a typo about 'maimly'. The others I mentioned may be finished in other PR.

**Comment by pissang** [comment]
Merged, @joyhchen thanks for the wonderful job!

**Pull request merged**